This package seems to specify an overly-restrictive version requirement 
on the `tatsu` dependency. The version it wants is more than two years
old and not packaged by conda-forge; meanwhile, things seem to work with
newer versions.

diff --git a/pyproject.toml b/pyproject.toml
index 5f38759f39..d4bc37e953 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -35,7 +35,7 @@ dependencies = [
     'beancount >= 2.3.4',
     'click > 7.0',
     'python-dateutil >= 2.6.0',
-    'tatsu >= 5.7.4, < 5.8.0',
+    'tatsu >= 5.7.4',
 ]
 
 [project.optional-dependencies]
